packages/api-client-next/README.md

```diff
@@ -4,7 +4,105 @@ Welcome to `@shopware/api-client` package.
 
 For getting started documentation visit [https://frontends.shopware.com/](https://frontends.shopware.com/)
 
-Documentation specific for this package: [api-client](https://frontends.shopware.com/packages/api-client.html)
+## Installation
```

Comment:
Please use https://github.com/shopware/frontends/blob/dbd34120a025efa20813ad631ea0b5b3f5f0fb2e/packages/composables/README.md as a template to make api-client-next's README similar to other packages :)

Reply:
thanks, good suggestion, applied